src/engineer/one_month_forecast.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import calendar
+from datetime import date
+from pathlib import Path
+import xarray as xr
+import warnings
+
+from typing import cast, Dict, Optional, Tuple
+
+from ..utils import minus_months
+from .engineer import Engineer
+
+
+class OneMonthForecastEngineer(Engineer):
+    def __init__(self, data_folder: Path = Path("data")) -> None:
+        self.name: str = "one_month_forecast"
+
+        super().__init__(self.name, data_folder)
+
+    @staticmethod
+    def stratify_xy(
+        ds: xr.Dataset,
+        year: int,
+        target_variable: str,
+        target_month: int,
+        pred_months: int,
+        expected_length: Optional[int],
```

Comment:
Can the default values be put back here too?

Reply:
absolutely!